collections/src/main/scala/strawman/collection/mutable/Builder.scala

```diff
@@ -309,4 +311,119 @@ class StringBuilder(private val sb: java.lang.StringBuilder) extends Builder[Cha
   def setLength(len: Int): Unit = sb.setLength(len)
 
   def update(idx: Int, elem: Char): Unit = sb.setCharAt(idx, elem)
+
+
+  /** Like reverse, but destructively updates the target StringBuilder.
+   *
+   *  @return   the reversed StringBuilder (same as the target StringBuilder)
+   */
+  def reverseContents(): StringBuilder = {
```

Comment:
reverseInPlace would be a more consistent name in the new design, and these methods now return `this.type`